src/components/radioGroup/radio/radio.tsx

```diff
@@ -0,0 +1,64 @@
+import * as RadioGroup from '@radix-ui/react-radio-group';
+import classNames from 'classnames';
+import type React from 'react';
+import { useId, type ComponentPropsWithoutRef } from 'react';
+import { Icon, IconType } from '../../icon';
+
+export interface IRadioProps extends ComponentPropsWithoutRef<'button'> {
+    /**
+     * Radio label
+     */
+    label: string;
+    /**
+     * Indicates whether the radio input is disabled
+     */
+    disabled?: boolean;
+    /**
+     * Value of the radio input
+     */
+    value: string;
+    /**
+     * Variant indicating the label positioning
+     * @default right
+     */
+    variant?: 'left' | 'right';
```

Comment:
I'm calling this as `labelPosition`, I think it better describe the property. Let me ping Selim to check if he's aligned